texts/puupankki/puupankki.kaz.conllu

```diff
@@ -5044,15 +5484,16 @@
 
 # sent_id = kdt.tagged.txt:109:1764
 # text = Сен бұл жолдан аман-есен қайтарсың, еліңе аман барарсың... — деді.
-1	Сен	сен	PRON	prn	Case=Nom|Number=Sing|Person=2|PronType=Prs	11	nsubj	_	_
+# labels = checked_IFS
+1	Сен	сен	PRON	prn	Case=Nom|Number=Sing|Person=2|PronType=Prs	7	nsubj	_	_
 2	бұл	бұл	DET	det	PronType=Dem	3	det	_	_
-3	жолдан	жол	NOUN	n	Case=Abl	7	nmod	_	_
-4	аман	аман	ADJ	adj	_	6	compound	_	SpaceAfter=No
+3	жолдан	жол	NOUN	n	Case=Abl	7	obl	_	_
+4	аман	аман	ADJ	adj	_	7	advmod	_	SpaceAfter=No
 5	-	-	PUNCT	guio	_	6	punct	_	SpaceAfter=No
-6	есен	есен	ADJ	adj	_	7	advmod	_	_
+6	есен	есен	ADJ	adj	_	4	compound	_	_
```

Comment:
This flip is debatable. The rule of thumb in UD seemes to be, all other things being equal, to make the first component the head (cf. conjunctions), hence this change probably.

Although in kk it is currently the other way  around in case of numerical compounds: https://universaldependencies.org/kk/dep/compound.html